doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,12 @@
   },
```

Comment:
<p>How about computing pairwise differences instead? Seeing this gave me the idea and shows cool xarray broadcasting and aligning:</p><p> </p><pre class="ql-syntax" spellcheck="false">idata.posterior.theta - idata.posterior.theta.rename(school="school_bis")
</pre><p> </p><p>Then, selecting <code>.sel(school="Choate", school_bis="Deerfield")</code>  gives the difference in effect between Choate and Deerfield, and likewise for all pairwise combinations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
hi! this was brilliant, v cool, i added it, thanks for suggesting!!